### PR TITLE
fix(web): fix typo + refactor empty state messages

### LINKED ISSUE #582: [BUG] web: Typo in `Draft` empty-state of Documents.
## Issue Description

There is a minor typo in the `empty-state.tsx` component for the `Draft` status:

typo-documenso

## Steps to Reproduce

- Go to dashboard of the app in staging environment.
- Check the `Draft` section.

## Expected Behavior

`then` should be changed to `the`

## Current Behavior

There is a minor typo in the `empty-state.tsx` component for the `Draft` status.

## Checklist

- [x] I have searched the existing issues to make sure this is not a duplicate.
- [x] I have provided steps to reproduce the issue.
- [x] I have included relevant environment information.
- [x] I have included any relevant screenshots.
- [x] I understand that this is a voluntary contribution and that there is no guarantee of resolution.

### PR DESCRIPTION
**Pull request by Bitnagar**
Fixes #582 

This PR fixes a typo and refactors empty state messages in `apps/web/src/app/(dashboard)/documents/empty-state.tsx` file for Documents in the Dashboard.

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by Bitnagar** [comment]
> Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:
> 
> https://documen.so/cla
@Mythie I did sign the CLA and filled the form before making the PR. 😄

**Comment by Bitnagar** [comment]
Hi @Mythie , @dguyen . Hope you're well. I understand you might be busy. I was just wondering when can i expect this PR to get merged? Appreciate your time and efforts. 😄

**Comment by ElTimuro** [comment]
image

File: apps/web/src/app/(dashboard)/documents/empty-state.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const EmptyDocumentState = ({ status }: EmptyDocumentProps) => {
     .otherwise(() => ({
       title: 'Nothing to do',
       message:
-        'All documents are currently actioned. Any new documents are sent or recieved they will start to appear here.',
+        'All documents have been processed. Any new documents that are sent or received will start to appear here.',
```
Comment (catalinpit) [inline apps/web/src/app/(dashboard)/documents/empty-state.tsx:35]: Can this be changed to `Any new documents that are sent or received will show here.',`, please?

**Review by catalinpit — CHANGES_REQUESTED** [review]

File: apps/web/src/app/(dashboard)/documents/empty-state.tsx (hunk 1 above)
Comment (Bitnagar) [reply apps/web/src/app/(dashboard)/documents/empty-state.tsx:35]: Should i replace the whole sentence? or just the `Any new document...` part?

Comment (catalinpit) [reply apps/web/src/app/(dashboard)/documents/empty-state.tsx:35]: @Bitnagar only the second one - `Any new documents...` please.

Comment (Bitnagar) [reply apps/web/src/app/(dashboard)/documents/empty-state.tsx]: Done. Let me know if any other change is there.

**Review by catalinpit — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**